wolox-training/ap-swift
# Rentals list

## Summary ##
**Trello card**: https://trello.com/c/WhvZvFzW/26-vista-para-libros-alquilados
**Trello card**: https://trello.com/c/widheS0P/25-wishlist

Whislist view and Rentals view

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press on Rentals tab
  * My Rentals tab should appear and load book cells
  * Press on Wishes tab
  * Wishes tab should appear and load book cells

## Screenshots ##

### iPhone 7

![rentals](https://cloud.githubusercontent.com/assets/25587882/24057038/b7927320-0b24-11e7-8b79-daf99194bfc7.png)

![whises](https://cloud.githubusercontent.com/assets/25587882/24057041/b9b2c376-0b24-11e7-8190-e739524e2f4d.png)


## Screencast ##

![rentals_wishes_list](https://cloud.githubusercontent.com/assets/25587882/24057114/00ee3996-0b25-11e7-834f-c7aa7ef10c33.gif)
